docs/user-guide/update-demo/local/index.html

```diff
@@ -23,11 +23,11 @@
 <body ng-controller="ButtonsCtrl">
   <div ng-repeat="server in servers" class="pod">
     <img src="/api/v1/proxy/namespaces/default/pods/{{server.podName}}/{{server.image}}" height="100px" width="100px" />
-    <b>ID:</b> {{server.podName}}<br>
-    <b>Host:</b> <a href="/api/v1/proxy/namespaces/default/pods/{{server.podName}}/data.json">{{server.host}}</a><br>
-    <b>Status:</b> {{server.status}}<br>
-    <b>Image:</b> {{server.dockerImage}}<br>
-    <b>Labels:</b>
+    <font size="2" color=#7D7D7D>ID<font color=black> {{server.podName}}<br>
```

Comment:
Much better way of doing this is:
```html
<div class="k8s-example-update-demo-card-section">
  <span class="k8s-title">ID</span> <span>{{server.podName}}</span>
</div>
<div class="k8s-example-update-demo-card-section">
  <span class="k8s-title">Host</span> 
  <a href="/api/v1/proxy/namespaces/default/pods/{{server.podName}}/data.json">{{server.host}}</a>
</div>
... all others ...
```

And then in CSS:
```css
.k8s-example-update-demo-card-section {
  color: "black";
}
.k8s-example-update-demo-card-section .k8s-title {
  color: "#7D7D7D";
}
// if you want to make the font bigger add this class to ID title, e.g., `<span class="k8s-title k8s-card-id">ID</span>`
.k8s-example-update-demo-card-section .k8s-card-id {
  font-size: bigger;
}
```



If you want me to explain why this is better, then let me know. If not (you just want to get this merged), then just follow my guidance :) It is good to have high quality examples anyway.